README.md

```diff
@@ -121,10 +121,10 @@ The output jdk is at `./build/linux-x86_64-normal-server-$DEBUG_LEVEL/jdk`.
 **Note:** The above `make` command will build what is known as the [`default` target or "exploded image"](https://github.com/openjdk/jdk11u/blob/master/doc/building.md#Running-make). This build is exclusively meant for developers who want quick and incremental builds to test changes. If you are planning on evaluating your build (be it performance, minimum heap, etc.), then it is *highly advised* to use the `images` target. The `default` target is the (roughly) minimal set of outputs required to run the built JDK and is not guaranteed to run all benchmarks. It may have bloated minimum heap values as well. The `images` target can be built like so:
 
 ```console
-$ make CONF=linux-x86_64-normal-server-release THIRD_PARTY_HEAP=$PWD/../mmtk-openjdk/openjdk images
```

Comment:
Could you add a line saying "For performance evaluation, we expect you are using the `release` build"? That was the original reason the `make` command was not using `$DEBUG_LEVEL` and directly using `release`. I agree that it should have been sign posted a bit better.

Reply:
Or better yet, mention it in the note directly above the `make` command.

Reply:
Does the same applies to the "Profile-Guided Optimized Build" section? 
If that is the case, then I should probably either:
1. undo all the changes (i.e. restore the hardcoded -release values) but add a clarifying note, or
2. add "DEBUG_LEVEL=release" before the `make` commands to make it more explicit.

What do you think?

Reply:
I think the first one is probably better. And yes, the PGO build is also generally used for evaluation only. The `images` (and PGO build) have extra steps involved in them which only make sense when you want to perform some kind of evaluation. In the `images` case, the `default` build (an unoptimized dev build) is used to bootstrap a production build and in the PGO case, the Rust compiler use execution profiles to inform inlining decisions.